WordPress/Classes/Utility/ImageURLs/NSString+BlavatarURL.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  NSString+BlavatarURL.swift
+//  WordPress
+//
+//  Created by Andrew McKnight on 11/22/16.
+//  Copyright © 2016 WordPress. All rights reserved.
+//
+
+import Foundation
+
+extension NSString
+{
+    func isBlavatarURL() -> Bool {
```

Comment:
I would rather have a single helper class with this and another helper methods than separate categories.
It will make dependencies more explicit, and won't extra responsibilities to a class that we don't own.